sentinelhub/geometry.py

```diff
@@ -221,25 +221,53 @@ def transform_bounds(self, crs, always_xy=True):
         bbox_geometry = bbox_geometry.transform(crs, always_xy=always_xy)
         return bbox_geometry.bbox
 
-    def buffer(self, buffer):
-        """Changes both BBox dimensions (width and height) by a percentage of size of each dimension. If number is
-        negative, the size will decrease. Returns a new instance of BBox object.
-
-        :param buffer: A percentage of BBox size change
-        :type buffer: float
-        :return: A new bounding box of buffered size
+    def buffer(self, relative=None, absolute=None):
```

Comment:
What signature would you propose? Maybe something like:

```python
def buffer(self, buffer, is_absolute=False):
```

Hm, I can't think of a good name for the boolean parameter. Because, when you call `buffer(10, is_absolute=False)` then it is not obvious to a reader that this means a relative buffer. On the other hand, if you call `buffer(relative=3.5)` or `buffer(absolute=(5, 2))` then it is very explicit and clear what kind of buffer you are doing.

Reply:
The options i see:
- Separate methods: `buffer` and  `absolute_buffer` (so we don't introduce code breaks, we can add `relative_buffer` and slowly deprecate `buffer`)
- Have `buffer(self, buffer, relative=True)`
- Have `buffer(self, buffer, mode='relative')`

I find it best to make the faulty state unreachable, runtime errors are costly.

Reply:
I'll have to sleep on this. But I'm not crazy about any of these. I find them either too verbose or not explicit enough.

Reply:
Unfortunately, the existing `buffer` method was for relative size - looking back, absolute buffer would make much more sense (aligned with `buffer` methods of other libs - geopandas, shapely...) 

I like the approaches 
 * `buffer(self, buffer, relative=True)` or
 * `buffer(self, buffer, mode='relative')`
 
 I'd prefer `buffer(self, buffer, relative='False')`, as this would be a step towards other APIs, but would break existing code.

Reply:
Changed to `buffer(self, buffer, relative=True)` as discussed.